src/index.html

```diff
@@ -6,21 +6,428 @@
+                                </div>
+                                <div class="detail_line">
+                                    <div class="detail_img_number">
+                                       <img src="./assets/area-icon.svg" alt="" class="detail_img">
+                                       <div class="detail_number area_detail">
+                                        <span class="detail_value">136</span>
+                                        <span class="detail_unit">M2</span>
+    
+                                       </div> 
+                                    </div>
+                                    <span class="detail_title">Area</span> 
+                                </div>
+                            
+                        </div>
+                        <div class="estate_price_status">
+                            <div class="estate_price">
+                                $630,000
+                            </div>
+                            <div class="estate_status">
+                                FOR RENT
+                            </div>
+                        </div>
+                    </div>
+                </li>
+                
+                
+            </ul>
+            <div class="option_buttons">
+                <button class="option_button">VIEW MORE SALE</button>
+                <button class="option_button">VIEW MORE RENT</button>
+            </div>
+        </div>
+        <footer class="general_footer">
+            <div class="footer_element  nav_contact">
+                <ul class="nav_menu2">
+                    <li class="nav_menu2_element">Home</li>
+                    <li class="nav_menu2_element">Buy</li>
+                    <li class="nav_menu2_element">Rent</li>
+                    <li class="nav_menu2_element">About Us</li>
+                    <li class="nav_menu2_element">News</li>
+                    <li class="nav_menu2_element">Contact</li>
+
+                </ul>
+                <div class="nav_contact_separator"></div>
+                <ul class="address_list">
+                    <li class="address_list_element">Easy House Renting Agency</li>
+                    <li class="address_list_element">Transaction / Gestion / Location</li>
+                    <li class="address_list_element">306 Rivera St, WellCity, BOLIVIA</li>
+                    <li class="address_list_element">+2135 89 6558 401 235</li>
+                    <li class="address_list_element">info@easyhouse.bl</li>
+                </ul>
+                <div class="nav_contact_separator"></div>
+                <div class="social_media_list">
+                    <img class="social_media_element" src="./assets/facebook-icon.svg" alt="">
+                    <img class="social_media_element" src="./assets/instagram-icon.svg" alt="">
+                    <img class="social_media_element" src="./assets/twitter-icon.svg" alt="">
```

Comment:
<img width="1349" alt="footer" src="https://github.com/Nelitaa/EasyHouse/assets/98427585/90a8dbd9-c6f2-4b83-8cfc-3c8a5184531f">
I've noticed that the images in a particular section of the code aren't displaying as expected. Please investigate and resolve this issue to ensure the images appear correctly in the specified section.